libs/openFrameworks/sound/ofFmodSoundPlayer.cpp

```diff
@@ -1,3 +1,7 @@
+#include "ofConstants.h"
+
+#ifdef OF_SOUND_PLAYER_FMOD
```

Comment:
can you remove this? sometimes we don't have that define but still want to use it optionally like in linux and this would prevent from doing so. i guess if we have the headers even if the library is not linked it should compile. also we can remove this file from the makefiles by excluding it for mingw 32bit